### DISCUSSION TITLE
How do I disable Alt+mouse drag window repositioning?

### DISCUSSION THREAD

**Discussion by MrDOS** [labwc/labwc#1318] [Q&A] [answered]
I'm not sure if this is actually implemented in Labwc or wlroots, but inheriting from the long-established X convention, in Labwc, I can hold Alt while clicking and dragging anywhere in a window to reposition it. Is there any way to disable this behaviour, or better, bind it to a different modifier key? I'm trying to use an application that has its own, un-rebindable Alt+drag behaviour.

**Comment by Consolatis** [comment, marked as answer]
Edit:
See comment below from MrDOS.

**Reply by ideasman42** [reply]
Tested this and it's not working, the only thing I found that works is to remove `<default />` but then none of the other basic bindings work for dragging windows for e.g.

**Reply by Consolatis** [reply]
This should still work, the defaults haven't changed.

**Reply by MrDOS** [reply]
Yeah, this is still effectively what I'm doing. I have:

```xml
<mouse>
    <default />

    <context name="Frame">
        <mousebind button="A-Left" action="Press" />
        <mousebind button="A-Left" action="Drag" />
        <mousebind button="A-Right" action="Press" />
        <mousebind button="A-Right" action="Drag" />

        <mousebind button="W-Left" action="Drag">
            <action name="Move" />
        </mousebind>
        <mousebind button="W-Right" action="Drag">
            <action name="Resize" />
        </mousebind>
    </context>
</mouse>
```

**Reply by ideasman42** [reply]
Ah, seems only overriding Drag isn't enough, thanks for the hint _(edit, this works, thanks!)._

**Comment by Consolatis** [comment]
An alternative solution would be to use the `ToggleKeybinds` [action](https://labwc.github.io/labwc-actions.5.html#entry_action_name=togglekeybinds), it should prevent all keybinds as long as that specific window has keyboard focus.

**Comment by MrDOS** [comment]
Thanks for the quick response. I somehow completely overlooked that default bind. `ToggleKeybinds` would have been a good alternative, but I think I'll get farther by retraining my muscle memory with Super as the modifier for both move and resize actions, so your first suggestion is perfect.
